tests/fuzz/README.md

```diff
@@ -50,6 +50,16 @@ See the help of the relevant command for options.
 Flags not parsed by `fuzz.py` are passed to the fuzzing engine.
 The command used to run the fuzzer is printed for debugging.
 
+Here's a helpful command to fuzz all targets in parallel,
+stopping only if a bug is found:
+```
+for target in $(./fuzz.py list); do
+    ./fuzz.py libfuzzer $target -jobs=10 -workers=10 -max_total_time=1000 || break;
```

Comment:
This doesn't run all the fuzzers in parallel. It runs one fuzzer at a time.

I'd suggest something like:

```
./fuzz.py list | xargs -P$(wc -l ./fuzz.py list) -I__ sh -c "./fuzz.py libfuzzer __ 2>&1 | tee __.log"
```

But double check that the command works first :)